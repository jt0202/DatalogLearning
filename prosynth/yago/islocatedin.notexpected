france_in_the_eurovision_song_contest_1960	lt_VirginiaVirginiagt
westerly	kuala_lumpur
miraxplaza_ukraine	dobrich_province
prairielands_council	lincolnshire
xtra_west	anantapur_district
