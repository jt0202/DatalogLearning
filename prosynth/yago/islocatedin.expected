ata_hussain_fani_chishti	gaya_india
camiguin	philippines
werner_van_den_valckert	amsterdam
gia_carangi	philadelphia
